src/main/java/org/junitpioneer/jupiter/EnvironmentVariableUtils.java

```diff
@@ -100,8 +101,8 @@ private static Map<String, String> getFieldValue(Class<?> clazz, Object object,
 			field.setAccessible(true); //NOSONAR illegal access required to implement the extension
 		}
 		catch (InaccessibleObjectException ex) {
-			throw new ExtensionConfigurationException(
-				"Cannot access Java runtime internals to modify environment variables. "
+			throw new PreconditionViolationException(
```

Comment:
What's with line 66? What exact situation handles that `catch`? Should this be a `PreconditionViolationException` as well?

Reply:
Yeah would keep them aligned.